tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -909,3 +909,84 @@ def softmax(x):
             result.append(DetectionPrediction(identifier, labels, det_scores, x_mins, y_mins, x_maxs, y_maxs))
 
         return result
+
+
+class UltraLightweightFaceDetectionAdapter(Adapter):
+    """
+    Class for converting output of Ultra-Lightweight Face Detection models to DetectionPrediction representation
+    """
+    __provider__ = 'ultra_lightweight_face_detection'
+
+    @classmethod
+    def validate_config(cls, config, fetch_only=False, **kwargs):
+        return super().validate_config(
+            config, fetch_only=fetch_only, on_extra_argument=ConfigValidator.ERROR_ON_EXTRA_ARGUMENT
+        )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'scores_out': StringField(description="Scores output layer name."),
+            'boxes_out': StringField(description="Boxes output layer name."),
+        })
+
+        return parameters
+
+    def configure(self):
+        self.scores_out = self.get_value_from_config('scores_out')
+        self.boxes_out = self.get_value_from_config('boxes_out')
+
+        # Set default values
+        self.prob_threshold = 0.7
+        self.iou_threshold = 0.5
+
+    def process(self, raw, identifiers, frame_meta):
+        raw_outputs = self._extract_predictions(raw, frame_meta)
+
+        batch_scores = raw_outputs[self.scores_out]
+        batch_boxes = raw_outputs[self.boxes_out]
+
+        result = []
+        for identifier, scores, boxes in zip(identifiers, batch_scores, batch_boxes):
+            detections = {'labels': [], 'scores': [], 'x_mins': [], 'y_mins': [], 'x_maxs': [], 'y_maxs': []}
+
+            for label, score in enumerate(np.transpose(scores)):
+                if label == 0:
+                    continue
+
+                mask = score > self.prob_threshold
+                filtered_boxes, filtered_score = boxes[mask, :], score[mask]
+                if filtered_score.size == 0:
+                    continue
+
+                x_mins = (filtered_boxes[:, 0])
+                y_mins = (filtered_boxes[:, 1])
+                x_maxs = (filtered_boxes[:, 2])
+                y_maxs = (filtered_boxes[:, 3])
```

Comment:
Why do you wrap output to the brackets?
```suggestion
                x_mins, y_mins, x_maxs, y_maxs = filtered_boxes.T
```

Reply:
Done